tests/publiccloud/storage_perf.pm

```diff
@@ -103,9 +104,14 @@ sub run {
 
     my $tags = {
         instance_type => get_required_var('PUBLIC_CLOUD_INSTANCE_TYPE'),
+        job_id => get_current_job_id(),
+        os_provider => get_var('PUBLIC_CLOUD_PROVIDER'),
```

Comment:
```suggestion
        os_provider => get_required_var('PUBLIC_CLOUD_PROVIDER'),
```
I think this tag is required to ensure that all collected measurements will have this tag.

Reply:
In this case also DB_URI will be required.

Reply:
ok, this last not applicable.